jekyll/_cci2/version-control-system-integration-overview.adoc

```diff
@@ -54,53 +55,55 @@ GitHub can be integrated into your CircleCI org in one of two ways, via the GitH
 | Feature | GitHub App | GitLab | GitLab Self-Managed | GitHub OAuth | Bitbucket Cloud | Bitbucket Data Center | CircleCI server
 
 | link:https://circleci.com/changelog/changes-to-code-checkout-for-orgs-that-integrate-with-github-app/[Secure HTTPS checkout]
-| icon:thumbs-up[role="circle-green"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
+| [.circle-green]#**Yes**#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
+| [.circle-green]#**Yes** ^2^#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
 
 | Reduced permissions when integrating with CircleCI. Select a subset of repositories to be used with CircleCI.
-| icon:thumbs-up[role="circle-green"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
+| [.circle-green]#**Yes**#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
+| [.circle-green]#**Yes** ^2^#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
 
 | link:https://discuss.circleci.com/t/circleci-config-suggestions-bot/47918[CircleCI config suggestions bot]
-| icon:thumbs-up[role="circle-green"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
+| [.circle-green]#**Yes**#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
+| [.circle-green]#**Yes** ^2^#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
 
 | xref:set-up-multiple-configuration-files-for-a-project#[Set up multiple configuration files for a project]
-| icon:thumbs-up[role="circle-green"]
-| icon:thumbs-down[role="circle-red"] ^1^
-| icon:thumbs-down[role="circle-red"] ^1^
-| icon:thumbs-down[role="circle-red"] ^1^
-| icon:thumbs-down[role="circle-red"] ^1^
-| icon:thumbs-down[role="circle-red"] ^1^
-| icon:thumbs-down[role="circle-red"] ^1^
+| [.circle-green]#**Yes**#
+| [.circle-red]#**No** ^1^#
+| [.circle-red]#**No** ^1^#
+| [.circle-green]#**Yes** ^1^, ^2^#
```

Comment:
can we try removing the comma or making sure it appears at the same height as the 1 and 2?